gita/__main__.py

```diff
@@ -79,6 +80,14 @@ def f_add(args: argparse.Namespace):
             if new_groups:
                 print(f'Created {len(new_groups)} new group(s).')
                 utils.write_to_groups_file(new_groups, 'a+')
+        if new_repos and args.group:
+            gname = args.group
+            if gname in groups:
```

Comment:
This check is unnecessary, line 411 already guarantees it.

Reply:
right, fixed